.env.sample

```diff
@@ -1,8 +1,8 @@
-# Configure your dev environment 
-DATABASE_PORT=<wanted postgres database port>
-DATABASE_USER=<wanted postgres database username>
-DATABASE_PASSWORD=<wanted postgres database password>
-
-DATABASE_NAME=connect_four_reboot_admin
-PGRST_DB_ANON_ROLE=web_anon
-PGRST_DB_SCHEMA=public
\ No newline at end of file
+## Please update those values if you modify the supabase default configuration
+SUPABASE_DB_URL=postgresql://postgres:postgres@127.0.0.1:54322/postgres ## supabase default url
+SUPABASE_USER=postgres ## supabase default user
+SUPABASE_PASSWORD=postgres ## supabase default password
+SUPABASE_DBNAME=postgres ## supabase default dbname
+SUPABASE_DOCKER_NAME=supabase_db_connect-four-reboot-admin ## supabase default docker_name
+VITE_SUPABASE_URL= http://127.0.0.1:54321 ## supabase url for react admin dataprovider
+VITE_SUPABASE_API_KEY= ## please put your supabase anon key
```

Comment:
The Supabase anon key for local environment is the same for everyone (it's not autogenerated), so you can safely commit it here